assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/DimensionTabs.js

```diff
@@ -16,6 +16,63 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import Tab from '@material/react-tab';
+import TabBar from '@material/react-tab-bar';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_FORMS } from '../../../../../googlesitekit/datastore/forms/constants';
+import { FORM_ALL_TRAFFIC_WIDGET } from '../../../datastore/constants';
+const { useSelect, useDispatch } = Data;
+
 export default function DimensionTabs() {
-	return null;
+	const dimensionName = useSelect( ( select ) => select( CORE_FORMS ).getValue( FORM_ALL_TRAFFIC_WIDGET, 'dimensionName' ) || 'ga:channelGrouping' );
```

Comment:
@kostyalmm this is not mentioned in the IB but let's get the `dimensionName` variable from the parent component which also pulls it from the datastore. Let's do it like it's done for `TotalUserCount` and `UserCountGraph` components.